src/scripts/migrate.py

```diff
@@ -0,0 +1,142 @@
+"""Slurp migration pipeline
+
+TODO's:
+  -
+"""
+import os.path
+from argparse import ArgumentParser
+from typing import Dict, List
+
+import pandas as pd
+import oaklib
```

Comment:
@matentzn Had some issues with (i) dependencies for `SqlImplementation` and also (ii) things not working while no internet connection. Opened up some issues / PRs: 
[OAK: [PR](https://github.com/INCATools/ontology-access-kit/pull/232), issues ([1](https://github.com/INCATools/ontology-access-kit/issues/234)); relation-graph: [PR](https://github.com/balhoff/relation-graph/pull/146); rdftab: [PR](https://github.com/ontodev/rdftab.rs/pull/22); prefix_commons: issues([1](https://github.com/prefixcommons/prefixcommons-py/issues/25)); [linkml-runtime issue](https://github.com/linkml/linkml-runtime/issues/198)]

Reply:
`ProntoImplementation` seems like it is more correct with the results it is returning and has more working methods available than `SqlImplementation`, so at this point I am using that.